R/run.R

```diff
@@ -191,6 +191,11 @@ log_image_to_run <- function(name, path = NULL, plot = NULL,
   if (is.null(run)) {
     run <- get_current_run()
   }
+  if (!is.null(plot)) {
+    path <- "_generated_rplot.png"
```

Comment:
it should be created under system standard temp directory and get cleaned up at the end.

Reply:
made the name consistent with python sdk.